WalletWasabi.Fluent/ApplicationStateManager.cs

```diff
@@ -126,7 +126,7 @@ private void CreateAndShowMainWindow()
 		_compositeDisposable = new();
 
 		Observable.FromEventPattern<CancelEventArgs>(result, nameof(result.Closing))
-			.Select(args => (args.EventArgs, !ApplicationViewModel.CanShutdown()))
+			.Select(args => (args.EventArgs, !ApplicationViewModel.CanShutdown(), ApplicationViewModel.MainViewCanShutdown()))
```

Comment:
passing `ApplicationViewModel.MainViewCanShutdown()` doesn't seem necessary.

Reply:
It's leftover, removed